examples/amm/src/contract.rs

```diff
@@ -303,7 +303,7 @@ impl Contract for AmmContract {
         }
     }
 
-    async fn finalize(&mut self) {
+    async fn store(mut self) {
         self.state.save().await.expect("Failed to save state");
```

Comment:
I wonder if we should rename `View::save`

Reply:
Yeah, I think that makes sense :thinking: